src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/CollectionCodecs.java

```diff
@@ -143,4 +211,22 @@ static JsonNode toJsonNode(
     }
     return result;
   }
+
+  /** Method that will convert from driver-provided CQL Map type into JSON output. */
+  static JsonNode toJsonNodeFromMap(
+      JSONCodec<?, ?> elementCodec0, ObjectMapper objectMapper, Object mapValue)
```

Comment:
just checking: we support multiple element codecs going toCQL and then only need one going toJson

Reply:
Exactly. It sort of suggests possible benefits of splitting `JSONCodec` into 2 parts (since one is many-to-many, another one-to-many). But for now there's some duplication and to-JSON case can basically pick any of matches.